docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header divider="none">
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane divider="none">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer divider="none">
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane on right
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Pane position="end">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without padding
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header padding="none">
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane padding="none">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content padding="none">
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer padding="none">
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without header or footer
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+</SplitPageLayout>
+```
+
+### With non-sticky pane
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane sticky={false}>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+## Props
+
+### SplitPageLayout
+
+<PropsTable>
+  <PropsTableSxRow />
+</PropsTable>
+
+### SplitPageLayout.Header
+
+<PropsTable>
+  <PropsTableRow
+    name="divider"
+    // TODO: Document ResponsiveValue
+    // This prop accepts a viewport range map (i.e. {regular: ..., narrow: ...}) in addition to a single value.
+    // The `regular` and `wide` keys of the viewport range map can be 'none' or 'line'.
+    // The `narrow` key of the viewport range map can be 'none' or 'line' or 'filled'.
+    type={`ResponsiveValue<
+  'none' | 'line',
+  'none' | 'line' | 'filled'
+>`}
```

Comment:
(for later): Can we bring responsive divider prop to PageLayout as well, i don't like `dividerWhenNarrow` 😅

Reply:
Yup. That's the plan!